cpp/include/cudf/scalar/scalar.hpp

```diff
@@ -89,7 +95,10 @@ class scalar {
   rmm::device_scalar<bool> _is_valid;  ///< Device bool signifying validity
 
   scalar() = delete;
-
+  /**
+   * @brief Move Constructor for scalar.
```

Comment:
I think we should capitalize "Move" because it's at the beginning of the line, but we don't need to capitalize "Constructor." Applies to several places in this PR.
```suggestion
   * @brief Move constructor for scalar.
```